src/toolchain/toolchain.ts

```diff
@@ -212,14 +213,40 @@ export class SwiftToolchain {
                 if (!version) {
                     throw Error("Info.plist is missing the XCTEST_VERSION key.");
                 }
-                return path.join(
-                    platformPath,
-                    "Developer",
-                    "Library",
-                    `XCTest-${version}`,
-                    "usr",
-                    "bin"
-                );
+
+                const swiftVersion = await this.getSwiftVersion(targetInfo);
```

Comment:
Given we have already calculated the swift version in `SwiftToolchain.create` we could pass it into the function instead of parsing `targetInfo` again

Reply:
🤷‍♂️ sure, can do.